privacyidea/lib/auditmodules/sqlaudit.py

```diff
@@ -405,14 +407,22 @@ def search(self, search_dict, page_size=15, page=1, sortorder="asc",
         auditIter = self.search_query(search_dict, page_size=page_size,
                                       page=page, sortorder=sortorder,
                                       timelimit=timelimit)
-        try:
-            le = next(auditIter)
-            while le:
```

Comment:
Why did you change the ``while le`` loop to a ``while True`` loop?
Would there be an issue with having a "double" loop condition of "while le" and "StopIteration"?

Reply:
My first approach was to catch the error for each individual `auditIter` entry. Unfortunately the first call to `next(auditIter)` throws the error _and_ stops the iteration.
And while looking through examples to call `next(auditIter)` only once, the  `while True:` loop seemed to be the most pythonic way.